wc.c

```diff
@@ -2,7 +2,23 @@
 #include <string.h>
 #include <ctype.h>
 
+
+void print_help() {
+    printf("wc [-c | -l | -w] <filename>\n");
+    printf("Options:\n");
+    printf("    -c:    Count characters in the file\n");
+    printf("    -w:     Count words in the file\n");
```

Comment:
This line (and the following 2 lines) have bad indent.

Reply:
Done.